<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像处理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/svg/img_source/image.svg</normaloff>:/svg/img_source/image.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>20</y>
      <width>1121</width>
      <height>881</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 242, 240);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_rotate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;黑体&quot;;

border-radius: 10px;
background-color: rgb(228, 179, 99);
color: rgb(95, 51, 98);
color: rgb(49, 54, 56);</string>
     </property>
     <property name="text">
      <string>旋转</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_translation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;黑体&quot;;
border-radius: 10px;
background-color: rgb(228, 179, 99);
color: rgb(49, 54, 56);</string>
     </property>
     <property name="text">
      <string>平移</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_match">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;黑体&quot;;
border-radius: 10px;
background-color: rgb(228, 179, 99);
color: rgb(49, 54, 56);</string>
     </property>
     <property name="text">
      <string>模板匹配</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-50</y>
      <width>1101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 227, 227);
background-color: rgb(244, 215, 215);
background-color: rgb(250, 245, 245);
border-radius:35px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSlider" name="horizontalSlider_angle">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>951</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal{ height: 10px; 
                            left: 10px; 
                            right: 10px; 
                            background: #dcdcdc; 
}

QSlider::handle:horizontal{ 
                             radius: 7px;
                             border-radius: 7px; 
                             border: 3px solid rgb(255, 255, 255);
                             width:  9px; 
                             margin: -5px -3px; 
                             background: rgb(255, 183, 120); 
} 

QSlider::sub-page:horizontalSlider:disabled {

                             background: #6c6c6c; 
                             border-color: #999;
                             border-radius: 2px;
} 

QSlider::add-page:horizontalSlider:disabled {
 
                             background: #ffffff;
                             border-color: #999;
                             border-radius: 2px;
}</string>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_function">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;黑体&quot;;
color: rgb(49, 54, 56);</string>
     </property>
     <property name="text">
      <string>旋转</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_x">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal{ height: 10px; 
                            left: 10px; 
                            right: 10px; 
                            background: #dcdcdc; 
}

QSlider::handle:horizontal{ 
                             radius: 7px;
                             border-radius: 7px; 
                             border: 3px solid rgb(255, 255, 255);
                             width:  9px; 
                             margin: -5px -3px; 
                             background: rgb(255, 183, 120); 
} 

QSlider::sub-page:horizontalSlider:disabled {

                             background: #6c6c6c; 
                             border-color: #999;
                             border-radius: 2px;
} 

QSlider::add-page:horizontalSlider:disabled {
 
                             background: #ffffff;
                             border-color: #999;
                             border-radius: 2px;
}</string>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_y">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>70</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal{ height: 10px; 
                            left: 10px; 
                            right: 10px; 
                            background: #dcdcdc; 
}

QSlider::handle:horizontal{ 
                             radius: 7px;
                             border-radius: 7px; 
                             border: 3px solid rgb(255, 255, 255);
                             width:  9px; 
                             margin: -5px -3px; 
                             background: rgb(255, 183, 120); 
} 

QSlider::sub-page:horizontalSlider:disabled {

                             background: #6c6c6c; 
                             border-color: #999;
                             border-radius: 2px;
} 

QSlider::add-page:horizontalSlider:disabled {
 
                             background: #ffffff;
                             border-color: #999;
                             border-radius: 2px;
}</string>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_match_run">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;黑体&quot;;
border-radius: 10px;
background-color: rgb(228, 179, 99);
color: rgb(49, 54, 56);</string>
     </property>
     <property name="text">
      <string>执行</string>
     </property>
    </widget>
    <zorder>label_function</zorder>
    <zorder>horizontalSlider_angle</zorder>
    <zorder>horizontalSlider_x</zorder>
    <zorder>horizontalSlider_y</zorder>
    <zorder>pushButton_match_run</zorder>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>481</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 227, 227);
background-color: rgb(239 ,100 ,97);
border-radius:15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_kong">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;黑体&quot;;
color: rgb(49, 54, 56);</string>
     </property>
     <property name="text">
      <string>空域</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_space">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>481</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-top-left-radius:15px;
border-top-right-radius:15px;
border-bottom-left-radius:15px;
border-bottom-right-radius:15px;
background-color: rgb(223, 218, 218);
background-color: rgb(232, 233, 235);

</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>481</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 227, 227);
background-color: rgb(239 ,100 ,97);
border-radius:15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_pin">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;黑体&quot;;
color: rgb(49, 54, 56);</string>
     </property>
     <property name="text">
      <string>频域</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
	
	font: 9pt &quot;黑体&quot;;
	background-color: rgb(255, 255, 255);
	border: 0px solid gray;
	border-radius: 10px;
	
	color: #494949;
	padding: 0px 5px;
}


QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 0px;
    border-left-color: darkgray;
    border-left-style: solid; 
    border-right-width: 8px;
	border-right-color: #e8e8e8;
    border-right-style: solid;
    border-top-right-radius: 20px; 
    border-bottom-right-radius: 20px;
	font: 9pt &quot;黑体&quot;;
	color: #494949;
}


QListView::item {
    background: #e8e8e8;
}
 
QListView::item:hover {
    background: #bebebe;
	font: 8pt &quot;Consolas&quot;;
	color: #494949;
}

QComboBox QAbstractItemView::item:selected
{	
    background-color: #bebebe;

}</string>
     </property>
     <item>
      <property name="text">
       <string>傅里叶变换</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>离散余弦变换</string>
      </property>
     </item>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_frequency">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>481</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-top-left-radius:15px;
border-top-right-radius:15px;
border-bottom-left-radius:15px;
border-bottom-right-radius:15px;
background-color: rgb(223, 218, 218);
background-color: rgb(232, 233, 235);

</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_match">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
	
	font: 9pt &quot;黑体&quot;;
	background-color: rgb(255, 255, 255);
	border: 0px solid gray;
	border-radius: 10px;
	
	color: #494949;
	padding: 0px 5px;
}


QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 0px;
    border-left-color: darkgray;
    border-left-style: solid; 
    border-right-width: 8px;
	border-right-color: #e8e8e8;
    border-right-style: solid;
    border-top-right-radius: 20px; 
    border-bottom-right-radius: 20px;
	font: 9pt &quot;黑体&quot;;
	color: #494949;
}


QListView::item {
    background: #e8e8e8;
}
 
QListView::item:hover {
    background: #bebebe;
	font: 8pt &quot;Consolas&quot;;
	color: #494949;
}

QComboBox QAbstractItemView::item:selected
{	
    background-color: #bebebe;

}</string>
     </property>
     <item>
      <property name="text">
       <string>书上</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>平方差匹配</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>相关匹配</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>相关系数匹配</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_match">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>271</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGraphicsView" name="graphicsView_match">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>211</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-top-left-radius:15px;
border-top-right-radius:15px;
border-bottom-left-radius:15px;
border-bottom-right-radius:15px;
background-color: rgb(250, 242, 240);
</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
